docs/src/main/paradox/index.md

```diff
@@ -59,11 +59,32 @@ Configure `slick`:
 
 ## Database Schema
 
-- @extref:[Postgres Schema](github:/core/src/test/resources/schema/postgres/postgres-schema.sql)
-- @extref:[MySQL Schema](github:/core/src/test/resources/schema/mysql/mysql-schema.sql)
-- @extref:[H2 Schema](github:/core/src/test/resources/schema/h2/h2-schema.sql)
-- @extref:[Oracle Schema](github:/core/src/test/resources/schema/oracle/oracle-schema.sql)
-- @extref:[SQL Server Schema](github:/core/src/test/resources/schema/sqlserver/sqlserver-schema.sql)
+- @extref:[Postgres Schema](github:/core/src/main/resources/schema/postgres/postgres-schema.sql)
+- @extref:[MySQL Schema](github:/core/src/main/resources/schema/mysql/mysql-schema.sql)
+- @extref:[H2 Schema](github:/core/src/main/resources/schema/h2/h2-schema.sql)
+- @extref:[Oracle Schema](github:/core/src/main/resources/schema/oracle/oracle-schema.sql)
+- @extref:[SQL Server Schema](github:/core/src/main/resources/schema/sqlserver/sqlserver-schema.sql)
+
+For testing purposes the journal and snapshot tables can be created programmatically  using the provided `SchemaUtils`.
+
+
+
+Scala
+:  ```scala
+import akka.persistence.jdbc.schema.scaladsl.SchemaUtils
+implicit val system: ActorSystem = ActorSystem("example")
+val done: Future[Done] = SchemaUtils.dropAndCreate()
+```
+
+Java
+:  ```java
+import akka.persistence.jdbc.schema.javadsl.SchemaUtils;
+ActorSystem actorSystem = ActorSystem.create("example")
+CompletionStage<Done> done = SchemaUtils.dropAndCreate(actorSystem)
```

Comment:
we need to start using imports from code snippets, but I'm explicitly leaving that for another occasion so we can move faster with this PR.